Thales/thales_safenet/_meta/fields.yml

```diff
@@ -0,0 +1,38 @@
+safenet.sourceid:
+  name: safenet.sourceid
+  type: keyword
+  description: "original event id"
+
+safenet.tenantid:
+  name: safenet.tenantid
```

Comment:
You might want to use something like
https://www.elastic.co/guide/en/ecs/current/ecs-cloud.html#field-cloud-instance-id